junit5/src/main/java/cz/xtf/junit5/annotations/SkipFor.java

```diff
@@ -16,9 +16,20 @@
 public @interface SkipFor {
 
 	/**
-	 * Name or regexp pattern matching name of the image. For example "eap73-openjdk11-openshift-rhel8" or "eap73-openjdk11-.*"
+	 * Name or regexp pattern matching name of the image.
+	 * For example "eap73-openjdk11-openshift-rhel8" or "eap73-openjdk11-.*".
+	 *
+	 * One of {@code dockerLabelName} or {@code name} must be present.
 	 */
-	String name();
+	String name() default "";
+
+	/**
+	 * Name or regexp pattern matching name of the docker image (docker labels -> name).
+	 * For example "eap73-openjdk11-openshift-rhel8" or "eap73-openjdk11-.*".
+	 *
+	 * One of {@code dockerLabelName} or {@code name} must be present.
+	 */
+	String dockerLabelName() default "";
```

Comment:
The same here. Could you paste here example name from labels, please?

Reply:
Please check.